scripts/lua/policies/security/clientSecret.lua

```diff
@@ -36,8 +36,9 @@ local REDIS_PASS = os.getenv("REDIS_PASS")
 --@return a string representation of what this looks like in redis :clientsecret:clientid:hashed secret
 function process(securityObj)
   local red = redis.init(REDIS_HOST, REDIS_PORT, REDIS_PASS, 1000)
```

Comment:
The init call here either starts up a new connection with redis, or uses one from a connection pool if there are any idle ones (current connection pool size is set to 100)